sdk-java/src/main/java/io/littlehorse/sdk/common/config/LHConfig.java

```diff
@@ -40,6 +41,9 @@ public class LHConfig extends ConfigBase {
     /** The Client Id. */
     public static final String CLIENT_ID_KEY = "LHC_CLIENT_ID";
 
+    /** The Tenant Id for this client, null will be used if not set */
```

Comment:
Do we want `null` or `default`? How do we handle `null`...in some cases, perhaps we can "infer" the tenant id from the principal.

For example, if Principal `alice` only has access to the `foo` Tenant, then does `alice` need to specify this header? Automatically inferring it would be nice, but it could also be dangerous, because if someone edits the `alice` principal to give the principal permissions to a new Tenant, then existing applications would break.

Reply:
I suggest keep this as `null`, in that way there is only one place (server side) that determine the tenant Id when the header is not present.